common/utils.js

```diff
@@ -1012,11 +1013,28 @@
         };
     }])
 
-    .factory("UiUtils", ['$document', '$log', function($document, $log) {
+    .factory("UiUtils", ['$document', '$log', 'dataFormats', function($document, $log, dataFormats) {
+
+        function versionToRelativeTS(versionAsMillis) {
```

Comment:
Just add a comment of what it's supposed to return. You can copy the issue description explaining the logic here.

Also since it's in the UiUtils, you can use a more general name for it. `humanizeTimestamp`  or something like that.